troubleshooting.md

```diff
@@ -1435,3 +1435,48 @@ Using the default `overlay` storage driver, a `COPY`, `ADD`, or an I/O intensive
 This could be caused by the child container using `fuse-overlayfs` for writing to `/var/lib/containers/storage`. Writes can be slow with `fuse-overlayfs`. The solution is to use the native `overlay` filesystem by using a local directory on the host system as a volume to `/var/lib/containers/storage` like so: `podman run --privileged --rm -it -v ./nested_storage:/var/lib/containers/storage parent:latest`. Ensure that the base image of `parent:latest` in this example has no contents in `/var/lib/containers/storage` in the image itself for this to work. Once using the native volume, the nested container should not fall back to `fuse-overlayfs` to write files and the nested build will complete much faster.
 
 If you don't have access to the parent run process, such as in a CI environment, then the second option is to change the storage driver to `vfs` in the parent image by changing changing this line in your `storage.conf` file: `driver = "vfs"`. You may have to run `podman system reset` for this to take effect. You know it's changed when `podman info |grep graphDriverName` outputs `graphDriverName: vfs`. This method is slower performance than using the volume method above but is significantly faster than `fuse-overlayfs`
+
+### 43) `podman run --userns=auto` fails with "Error: creating container storage: not enough unused IDs in user namespace"
+
+If a container is already running with `--userns=keep-id`, then it's not possible to start another container with `--userns=auto`
+
+#### Symptom
+
+On a Linux system
+
+1. Create a new user
+   ```
+   $ sudo useradd test
+   ```
+2. Start a login session
+   ```
+   $ sudo machinectl shell --uid=test
+   ```
```

Comment:
I think this should be dropped as it just adds noise without real benefit

Reply:
Now fixed